### DISCUSSION TITLE
Why is the notebook layout not working ?

### DISCUSSION THREAD

**Discussion by Gitstar-OC** [fuma-nama/fumadocs#1114] [Q&A]
```jsx
import { DocsLayout } from 'fumadocs-ui/layouts/notebook';
import { baseOptions } from '@/app/layout.config';
import type { ReactNode } from 'react';
 
export default function Layout({ children }: { children: ReactNode }) {
  return (
    <DocsLayout {...baseOptions} tree={source.pageTree}>
      {children}
    </DocsLayout>
  );
}
```

This is what the above code is looking like 

https://github.com/user-attachments/assets/d330b1df-f82b-4bbd-9d5a-8794c6c2bbde

If I changed the layout back to normal it works fine. 

![image](https://github.com/user-attachments/assets/a6f3f6da-89cc-498d-96f4-0205393f3dcd)

What could be the reasor for it ? I have changed and modified the components a little bit and deleted some and replaced them with original shadcn components like button.

**Comment by Gitstar-OC** [comment]
![image](https://github.com/user-attachments/assets/ba0c1e3b-5150-4ce3-8904-34a2a888c745)

Could this be due to the error in the docs section of the components

**Comment by fuma-nama** [comment]
sorry for late, mind to provide a minimal reproduction?

**Reply by Gitstar-OC** [reply]
Sure, let me push the repo on github

**Comment by Gitstar-OC** [comment]
https://github.com/Gitstar-OC/notebook this is the repo url, just run the below commands 
```
git clone https://github.com/Gitstar-OC/notebook.git
cd notebook
```
```
npm install
npm run dev
```

And head over to http://localhost:3000/docs/

**Comment by Gitstar-OC** [comment]
@fuma-nama is it solved or I should propose a new pr change that will has the type of layout with simple navbar at the top the sidebar will still be there but it will look like the below image. 

![image](https://github.com/user-attachments/assets/976698f6-50f4-42fa-b0ad-b10ebf7ff6ba)

**Comment by Gitstar-OC** [comment]
Update: I created a new docs application in which the thing is working properly. The problem in the reproduction I gave you is that some styling and components that you used are modified versions of shadcn ui library so what happens is they conflict and as a result the styling gets affected. 

![image](https://github.com/user-attachments/assets/8597914e-3981-40c2-b807-ad46d177da7f)
